tests/requirements.yml

```diff
@@ -1,8 +1,13 @@
 ---
 - src: idealista.java-role
-  version: 2.0.0
+  version: 3.0.0
   name: java
 
-- src: idealista.zookeeper-role
-  version: 1.1.0
-  name: zookeeper
+#- src: idealista.zookeeper-role
```

Comment:
Guess we have to release new zookeeper role version, haven't we?

Reply:
Although it should not be done, please do not leave commented lines